src/emulator/host/include/host/config.h

```diff
@@ -40,6 +40,7 @@ struct Config {
     bool log_uniforms = false;
     std::vector<std::string> lle_modules;
     bool pstv_mode = false;
+    bool show_gui = false;
```

Comment:
```suggestion
    bool show_gui = true;
```

Reply:
same for here, like i have said, gui hide up part game, so hide it by defaut is normal